app/src/main/java/com/keylesspalace/tusky/view/ExtendedViewPager.kt

```diff
@@ -0,0 +1,32 @@
+package com.keylesspalace.tusky.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.viewpager.widget.ViewPager
+
+
+class ExtendedViewPager : ViewPager {
+    private var swipeable = true
+
+    constructor(context: Context) : this(context, null)
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(context, attrs)
+    }
+
+    private fun init(context: Context, attrs: AttributeSet?) {
+        val a = context.obtainStyledAttributes(attrs, com.keylesspalace.tusky.R.styleable.ExtendedViewPager)
+        try {
+            swipeable = a.getBoolean(com.keylesspalace.tusky.R.styleable.ExtendedViewPager_swipeable, true)
```

Comment:
Why make this styleable when it is never swipeable anyway? I would just call it `NoSwipeViewPager` and be done with it

Reply:
Done